### PR TITLE
docs: adding single project usages + Platform-specific customization + SVG vs PNG Image use

### PR DESCRIPTION
**Pull request by Kunal22shah**
Fixes: #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the new behavior?

Updated docs with single project usages, Platform-specific customization and SVG vs PNG Image use( #260 , #261 , [#15869](https://github.com/unoplatform/uno/issues/15869) )

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/using-uno-resizetizer.md [hunk 1]
```diff
@@ -95,6 +95,36 @@ During the creation of your `svg` file, please remember to make the `ViewBox` bi
 
 ### 4. Configuring the project to use generated app icon
 
+# [**Single Project Based Solution**](#tab/singleproject)
+In the single project structure, you will configure your app icon directly in the .csproj file, streamlining the setup across all platforms.
+
+* Create an `Icons` folder within your project under the `Assets` directory and add your icon files there. For example, place your `iconapp.svg` and its corresponding `appconfig.svg` in the `Assets\Icons` folder.
+* Open your `MyApp.csproj` file and add the following XML block to include the UnoIcon directly, which simplifies managing multiple project heads:
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:102]: By default this is done for you. If you create a new Uno app you'll notice there is alread an Assets/Icons folder and it contains a `icon.svg` and `icon_foreground.svg`. And the `<UnoIcon ... />` is included in the SDK so it doesn't need to explicitly be added to the .csproj.

So people can either replace those files, keeping the same name, or use the SDK properties to customize the files/names/properties.

File: doc/using-uno-resizetizer.md [hunk 2]
```diff
@@ -95,6 +95,36 @@ During the creation of your `svg` file, please remember to make the `ViewBox` bi
...
+This configuration automatically applies across all target platforms included in the single project structure.
+## Utilizing SDK Properties
+For basic adjustments, such as changing the icon's foreground color or applying a common modification across platforms, you can use SDK properties. This method is simpler and reduces the need for extensive XML configurations:
+
+```xml
+<PropertyGroup>
+    <UnoIconForegroundFile>Assets\Icons\appconfig.svg</UnoIconForegroundFile>
+    <UnoIconColor>#FF0000</UnoIconColor>
+</PropertyGroup>
+```
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:120]: Move this example usage to be after the listing of available properties

File: doc/using-uno-resizetizer.md [hunk 3]
```diff
@@ -217,6 +247,33 @@ Next, some adjustments are needed on `Android`, `Windows (WinUI)`, `WebAssembly`
...
+*  Create a `SplashScreen` folder within the `Assets` directory of your project and add your splash screen files there. For example, you might include a `splash_screen.svg` file in the Assets\SplashScreen folder.
+* Open your `MyApp.csproj` file and add the following XML block. This setup will apply the splash screen configuration to all platforms included in the single project:
+
+```xml
+<ItemGroup>
+    <UnoSplashScreen Include="Assets\SplashScreen\splash_screen.svg"
+                     BaseSize="128,128"
+                     Color="#512BD4"/>
+</ItemGroup>
+```
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:260]: Same comment as above, we should point out that these files, with their default names, and folder already exist by default in a new Uno app

File: doc/using-uno-resizetizer.md [hunk 4]
```diff
@@ -217,6 +247,33 @@ Next, some adjustments are needed on `Android`, `Windows (WinUI)`, `WebAssembly`
...
+## Utilizing SDK Properties
+* To facilitate easier customization, such as adjusting the base size or color of the splash screen, you can leverage SDK properties:
+```xml
+<PropertyGroup>
+    <UnoSplashScreenFile>Assets\SplashScreen\splash_screen.svg</UnoSplashScreenFile>
+    <UnoSplashScreenBaseSize>128,128</UnoSplashScreenBaseSize>
+    <UnoSplashScreenColor>#512BD4</UnoSplashScreenColor>
+</PropertyGroup>
+```
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:269]: Move example beneath the list of available properties

File: doc/using-uno-resizetizer.md [hunk 5]
```diff
@@ -95,6 +95,30 @@ During the creation of your `svg` file, please remember to make the `ViewBox` bi
 
 ### 4. Configuring the project to use generated app icon
 
+# [**Single Project Based Solution**](#tab/singleproject)
+
+* When you create a new Uno Platform application, an `Icons` folder is automatically generated under the `Assets` directory. This folder contains `icon.svg` and `icon_foreground.svg` files.
+* You can simply replace these files with your custom icons while retaining the file names, or you can customize the icon configuration using SDK properties if different names or additional configurations are needed.
+* This configuration automatically applies across all target platforms included in the single project structure.
+## Utilizing SDK Properties
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:103]:
```suggestion

## Utilizing SDK Properties
```

File: doc/using-uno-resizetizer.md [hunk 6]
```diff
@@ -95,6 +95,30 @@ During the creation of your `svg` file, please remember to make the `ViewBox` bi
...
+
+The Uno Platform SDK exposes several properties that simplify the customization of your app icon. These properties allow you to easily adjust key aspects like the base size, color, and icon files without detailed XML changes, making your development process more streamlined.
+
+* `UnoIconBackgroundFile`: Sets the background image file for the icon.
+* `UnoIconForegroundFile`: Sets the foreground image file for the icon.
+* `UnoIconForegroundScale`: Adjusts the scaling of the icon's foreground.
+* `UnoIconBackgroundColor`: Sets the background color of the icon.
+
+For basic adjustments, such as changing the icon's foreground color or applying a common modification across platforms, you can use SDK properties. This method is simpler and reduces the need for extensive XML configurations:
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:112]:
```suggestion
For basic adjustments, such as changing the icon's foreground color or applying a common modification across platforms, you can use SDK properties:
```

File: doc/using-uno-resizetizer.md [hunk 7]
```diff
@@ -95,6 +95,30 @@ During the creation of your `svg` file, please remember to make the `ViewBox` bi
...
+
+```xml
+<PropertyGroup>
+    <UnoIconForegroundFile>Assets\Icons\appconfig.svg</UnoIconForegroundFile>
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:116]:
```suggestion
    <UnoIconForegroundFile>Assets\Icons\customicon.svg</UnoIconForegroundFile>
```

File: doc/using-uno-resizetizer.md [hunk 8]
```diff
@@ -217,6 +241,28 @@ Next, some adjustments are needed on `Android`, `Windows (WinUI)`, `WebAssembly`
...
+The Uno Platform SDK exposes several properties that simplify the customization of your splash screen. These properties allow you to easily adjust key aspects like the base size, color, and icon files without detailed XML changes, making your development process more streamlined.
+
+* `UnoSplashScreenFile`: Specifies the image file for the splash screen.
+* `UnoSplashScreenBaseSize`: Sets the base size for the splash screen image.
+* `UnoSplashScreenColor`: Determines the background color of the splash screen.
+
+To facilitate easier customization, such as adjusting the base size or color of the splash screen, you can leverage SDK properties:
+```xml
+<PropertyGroup>
+    <UnoSplashScreenFile>Assets\SplashScreen\splash_screen.svg</UnoSplashScreenFile>
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:259]:
```suggestion
    <UnoSplashScreenFile>Assets\SplashScreen\custom_splash_screen.svg</UnoSplashScreenFile>
```

File: doc/using-uno-resizetizer.md [hunk 9]
```diff
@@ -312,7 +358,74 @@ Next, some adjustments are needed on `Android`, `Windows`, and `iOS`.
...
+```xml
+<PropertyGroup>
+    <!-- Default background color -->
+    <UnoIconBackgroundColor>#FFFFFF</UnoIconBackgroundColor>
+
+    <!-- Platform-specific overrides -->
+    <UnoIconBackgroundColor Condition="'$(TargetFramework)' == 'net6.0-android'">#000000</UnoIconBackgroundColor>
+    <UnoIconBackgroundColor Condition="'$(TargetFramework)' == 'net6.0-ios'">#FF0000</UnoIconBackgroundColor>
+    <UnoIconBackgroundColor Condition="'$(TargetFramework)' == 'net6.0-windows10.0.19041.0'">Transparent</UnoIconBackgroundColor>
+    <UnoIconBackgroundColor Condition="'$(TargetFramework)' == 'net6.0-wasm'">Transparent</UnoIconBackgroundColor>
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:375]: these should either be net8.0- or you can use the platform properties from the uno sdk:

https://github.com/unoplatform/uno/blob/58ddd2f113cd4d9524e0c5a0e13d928e1aa54fa8/src/Uno.Sdk/targets/Uno.IsPlatform.props#L3-L10

I'd rather that so we aren't out-of-date whenever we bump .net versions

Those should be able to be used here, right @jeromelaban?

File: doc/using-uno-resizetizer.md [hunk 10]
```diff
@@ -312,7 +358,74 @@ Next, some adjustments are needed on `Android`, `Windows`, and `iOS`.
 > Feel free to delete old assets related to the splash screen.
 
 -----
+## Platform-Specific Customization
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:361]: Make sure to walk through the document and ensure we have a line of whitespace above and below each heading.

```suggestion

## Platform-Specific Customization

```

File: doc/using-uno-resizetizer.md [hunk 11]
```diff
@@ -312,7 +358,74 @@ Next, some adjustments are needed on `Android`, `Windows`, and `iOS`.
...
+    <AndroidForegroundScale Condition="'$(TargetFramework)' == 'net6.0-android'">0.6</AndroidForegroundScale>
+    <WasmForegroundScale Condition="'$(TargetFramework)' == 'net6.0-wasm'">0.4</WasmForegroundScale>
+    <WindowsForegroundScale Condition="'$(TargetFramework)' == 'net6.0-windows10.0.19041.0'">0.3</WindowsForegroundScale>
+    <IOSForegroundScale Condition="'$(TargetFramework)' == 'net6.0-ios'">0.55</IOSForegroundScale>
+</PropertyGroup>
+```
+## Using SVG Images vs PNG Images with SVG underneath
+The Uno Platform allows for flexible image handling through direct SVG use or through asset generation via Uno.Resizetizer. Understanding when to use each approach can optimize your app's performance and visual quality.
+
+#### Direct SVG Usage
```
Comment (kazo0) [inline doc/using-uno-resizetizer.md:398]:
```suggestion
### Direct SVG Usage

```

Make sure to increment the number of `#` according to the parent header that you are under. Same for the rest of the document

**Review by agneszitte — APPROVED** [review]

**Pull request merged**